api.py

```diff
@@ -105,19 +107,30 @@ def post():
 # time must be a unix timestamp whereas for the other endpoints we require a UTC string. the reason for this is here we need to calculate with time whereas in the other endpoints we only hand it over to the database (which does not accept a unix timestamp)
 @app.route('/input/bulk.json', methods=['GET','POST'])
 def bulk():
-    data = request.form['data']
+
+
+    # Check if compressesed
+    if request.headers.get('Content-Encoding', None) == 'gzip':
+        data= zlib.decompress(request.get_data())
+        logging.debug("decompressed %s "%data)
+    else:
+        data = request.get_data()
```

Comment:
I think we should move this in a `before_request` method that it applies to all the endpoints.